cluster/gce/configure-vm.sh

```diff
@@ -968,7 +968,7 @@ function salt-node-role() {
 grains:
   roles:
     - kubernetes-pool
-  cbr-cidr: 10.123.45.0/30
+  cbr-cidr: 10.123.45.0/29
```

Comment:
a better thing to do in the longer term might be to just bite off a chunk of the cluster-cidr, at least we're sure we own that